README.md

```diff
@@ -19,6 +19,8 @@ for a production deployment, but a great fit for development and testing:_
   is none of the typical six seconds block time associated with `grandpa` or `aura`.
 * _If no CLI arguments are passed the node is started in development mode
   by default._
+* A custom logging filter is applied by default that hides block production noise and prints the contracts debug
+  buffer to the console.
```

Comment:
```suggestion
* A custom logging filter is applied by default that hides block production noise
  and prints the contracts debug buffer to the console.
```

Just because we managed to stay at roughly the 80 CPL so far in this file.